apps/server/src/shared/repo/schoolexternaltool/school-external-tool.repo.ts

```diff
@@ -38,6 +38,11 @@ export class SchoolExternalToolRepo extends BaseDORepo<
 		return domainObjects;
 	}
 
+	async countSchoolExternalToolsByExternalToolId(toolId: string): Promise<number> {
```

Comment:
do you really need this function, when you already get the list with find in your service. you could just use `.length`